hikari/internal/deprecation.py

```diff
@@ -33,12 +33,14 @@
 from hikari.internal import ux
 
 
-def warn_deprecated(name: str, /, *, removal_version: str, additional_info: str, stack_level: int = 3) -> None:
+def warn_deprecated(
```

Comment:
IMO these changes should be in a separate PR

Reply:
These will be reverted and replaced by #1473

Reply:
Ah I should've spotted that. My bad!